spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -55,7 +55,7 @@
       fill_in t('idv.form.password'), with: Features::SessionHelper::VALID_PASSWORD
       click_continue
 
-      expect(current_path).to be_in([idv_personal_key_path, idv_app_path])
+      expect(current_path).to be_in([idv_personal_key_path])
```

Comment:
Let's change these to an equals check, since we now only support the one.

```suggestion
      expect(current_path).to eq idv_personal_key_path
```